.code-samples.meilisearch.yaml

```
@@ -3,6 +3,42 @@
 # the documentation on build
 # You can read more on https://github.com/meilisearch/documentation/tree/master/.vuepress/code-samples
 ---
+get_all_tasks_filtering_1: |-
+  client.index("movies").getTasks() { result in
```

To keep consistency with the other code samples I would suggest we do this here but you can chose

```suggestion
  client.getTasks(params: TasksQuery(indexUid: ["movies"])) { result in
```